src/utils/private.ts

```diff
@@ -150,6 +150,8 @@ export const getWideConfig = async (context: Context) => {
     promotionComment: mergedConfigData["promotion-comment"],
     registerWalletWithVerification: mergedConfigData["register-wallet-with-verification"],
     enableAccessControl: mergedConfigData["enable-access-control"],
+    openAIKey: process.env.OPENAI_API_KEY || mergedConfigData["openai-api-key"],
```

Comment:
Pls remove `process.env.OPENAI_API_KEY`, reading from the bot's config is sufficient

Reply:
I'm confused, if we remove it from here then it's reliant on an API key existing somewhere in the repo or org config. Is this intended?

Reply:
The bot's config is hierachial (the latter rewrites the former):
1. [Default bot's config](https://github.com/ubiquity/ubiquibot/blob/development/ubiquibot-config-default.json) for all partners 
2. Partner's organization wide config (which by convention must be a private repository, [example](https://github.com/rndquu-org/ubiquibot-config/blob/55d8bef8fa07fd0adc23c5ab921e027368fdbd8d/.github/ubiquibot-config.yml))
3. Partner's repository wide config ([example](https://github.com/rndquu-org/test-repo/blob/main/.github/ubiquibot-config.yml))

So the `openai-api-key` is meant to be set in a private organization wide config

Reply:
Ah understood! Thanks for clearing that up